templates/index.html

```diff
@@ -26,7 +26,7 @@
 <meta property="og:image:width" content="1200"/>
 <meta property="og:image:height" content="630"/>
 
-<title>Hello AWS App Runner</title>
+<title>Hello AWS App Runner V2</title>
```

Comment:
I don't know that we need to update the title. Let's leave it as is and just update requirements.

Reply:
Agree ! You can ignore that last commit.

Reply:
You'll have to revert that commit for me to merge

Reply:
Reverted.